src/main/java/uk/gov/ons/ctp/integration/contactcentresvc/service/addressindex/model/AddressIndexSearchResultsDTO.java

```diff
@@ -0,0 +1,18 @@
+package uk.gov.ons.ctp.integration.contactcentresvc.service.addressindex.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import java.util.ArrayList;
+import lombok.Data;
+
+@Data
+@JsonIgnoreProperties({"apiVersion"})
```

Comment:
needed? won't Jackson just serialise what it can and dump the rest?

Reply:
Fraid not. 
I was wrong when I said that Jackson ignores json data that is not in the DTO. You must tell Jackson either to ignore individual fields or to ignore all non matching data.